questionnaire/2020/questions.yaml

```diff
@@ -403,19 +406,22 @@
     id: trend-tools
     responses: 
     - { text: Ansible, id: ansible }
+    - { text: Terraformm, id: terraform }
     - { text: Saltstack, id: saltstack }
     - { text: NAPALM, id: napalm }
     - { text: Nornir, id: nornir }
     - { text: Netbox, id: netbox }
     - { text: ELK (Logstash/Kibana), id: elk }
-    - { text: Grafana/Prometheus/Influxdb, id: tsdb }
+    - { text: Grafana, id: grafana }
+    - { text: Prometheus, id: prometheus }
+    - { text: Influxdb, id: influxdb }
```

Comment:
We're listing these tools under the new Streaming Telemetry section, I don't think we need to list them here also.